src/app/zap-templates/templates/app/access.zapt

```diff
@@ -0,0 +1,425 @@
+{{> header}}
+
+// Prevent multiple inclusion
+#pragma once
+
+#define GENERATED_ACCESS_PRIVILEGE__VIEW (0)
+#define GENERATED_ACCESS_PRIVILEGE__OPERATE (1)
+#define GENERATED_ACCESS_PRIVILEGE__MANAGE (2)
+#define GENERATED_ACCESS_PRIVILEGE__ADMINISTER (3)
+
+////////////////////////////////////////////////////////////////////////////////
+
+{{#all_user_clusters}}
```

Comment:
Should probably then condition on `{{#if (isServer clusterSide)}}` because ACL stuff makes no sense for client clusters, right?  That would address the bridge/OTA issue.

Similar for all the other loops over clusters here.

Reply:
Yep that's my next step, just had to step out for half an hour to get kids bikes ready since it's suddenly 17 deg celsius today.

Reply:
Switched to use `chip_server_clusters`.